KmsClient.js

```diff
@@ -503,26 +502,24 @@ export class KmsClient {
       capability = _getRootZcapId({keyId});
     }
 
+    const headers = await signCapabilityInvocation({
```

Comment:
Now this should be called within the try/catch so any errors thrown inside will be wrapped by the top level error (in all functions).

Reply:
Wait, but the reason I separated those calls to outside the try/catch, is to differentiate the errors that happen during zcap invocation, from those that happen during the http client operation to the webkms. I think calling `signCapabilityInvocation` from within the try/catch will lose some of that info.

Reply:
The information should never be lost, that's the point of the `cause` property -- to preserve the whole chain. By moving the code into the try/catch we get a top level error (you were trying to do X) and a cause (it failed for reason Y). Seeing that you were trying to KMS operation X and it failed because of signing a zcap invocation Y is better than only getting "you failed to sign zcap invocation Y".

Reply:
@dlongley - No prob, I moved all the `signCapabilityInvocation()` calls inside of the try/catch.